### PR TITLE
Add CDLA-Permissive-2.0

### LINKED ISSUE #1282: New license request: CDLA-Permissive-2.0 [SPDX-Online-Tools]
**1.** License Name: Community Data License Agreement Permissive 2.0
**2.** Short identifier: CDLA-Permissive-2.0
**3.** License Author or steward: The Linux Foundation
**4.** Comments: More examples of data sets using CDLA-Permissive-2.0 are listed in the LF's announcement at: https://linuxfoundation.org/press-release/enabling-easier-collaboration-on-open-data-for-ai-and-ml-with-cdla-permissive-20/

This is the successor to CDLA-Permissive-1.0, which is already on the SPDX License List: https://spdx.org/licenses/CDLA-Permissive-1.0.html

Also something of a successor in spirit to O-UDA-1.0: https://spdx.org/licenses/O-UDA-1.0.html

I'll follow up with a comment in the license-list-XML repo with my thoughts about this one and the License Inclusion Principles, so that I'm not asking others to do the work here  :)
**5.** Standard License Header: N/A
**6.** License Request Url: http://tools.spdx.org/app/license_requests/115
**7.** URL(s): https://cdla.dev/permissive-2-0/
**8.** OSI Status: Not Submitted
**9.** Example Projects: https://developer.ibm.com/exchanges/data/all/project-codenet/, https://msropendata.com/datasets/0a83fb6f-a759-4a17-aaa2-fbac84577318, https://msropendata.com/datasets/feed8996-d9d4-47b9-9938-44ed298628fc

### PR DESCRIPTION
**Pull request by swinslow**
Fixes #1282 

Signed-off-by: Steve Winslow <steve@swinslow.net>

### CODE REVIEW

**Comment by swinslow** [comment]
cc @jlovejoy @seabass-labrax @bsdimp, grateful if you (or others) can give a +1 for this PR  :)

File: src/CDLA-Permissive-2.0.xml [hunk 1]
```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+    <license licenseId="CDLA-Permissive-2.0"
+        name="Community Data License Agreement Permissive 2.0"
+        listVersionAdded="3.14">
```
Comment (seabass-labrax) [inline src/CDLA-Permissive-2.0.xml:5]:
```suggestion
        isOsiApproved="false" listVersionAdded="3.14">
```

**Review by seabass-labrax — APPROVED** [review]
Apart from needing the `isOsiApproved` attribute, it all looks good to me! :)

File: src/CDLA-Permissive-2.0.xml (hunk 1 above)
Comment (swinslow) [reply src/CDLA-Permissive-2.0.xml]: oops -- great catch, thanks @seabass-labrax  :)  now fixed!

**Pull request merged**